tests/testProvenanceSetup.py

```diff
@@ -132,15 +136,15 @@ def testAddCmd(self):
         self.assertEquals(cmds[0][0], "recProv.py")
         self.assertEquals(cmds[0][1], "-v")
         self.assertEquals(cmds[0][2], "4")
-        
+
         self.setup.addWorkflowRecordCmd("dbingest.py", "-u ray".split(),
                                         "/usr/local/bin/dbingest.py")
```

Comment:
Is that really what it's doing?  It appears to me that it's just testing adding a command, not actually executing anything there.